include/dlaf/communication/error.h

```diff
@@ -11,36 +11,36 @@
 #pragma once
 
 /// @file
+//
+#include "dlaf/common/utils.h"
+
+#include <mpi.h>
```

Comment:
Following [coding style](https://github.com/eth-cscs/DLA-Future/blob/master/misc/coding.md#includes), these headers must come after the C++ standard library ones.